_episodes/01-intro.md

```diff
@@ -59,98 +59,61 @@ loops back and waits for you to enter another command.
 stand alone tasks are usually refered to as commands.
 The most popular Unix shell is Bash, (the Bourne Again SHell --- so-called because
 it's derived from a shell written by Stephen Bourne).
 Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
-### What does it look like?
 
-A typical shell window looks something like:
+When the shell is first opened, you are presented with a **prompt**,
+indicating that the shell is waiting for input.
 
 ~~~
-bash-3.2$
-bash-3.2$ ls -F /
-Applications/         System/
-Library/              Users/
-Network/              Volumes/
-bash-3.2$
+$
 ~~~
+{: .language-bash}
 
-The first line shows only a **prompt**,
-indicating that the shell is waiting for input.
-Your shell may use different text for the prompt. Most importantly:
+The shell typically uses `$ ` as the prompt, but may use a different symbol.
+Most importantly:
 when typing commands, either from these lessons or from other sources,
 *do not type the prompt*, only the commands that follow it.
 
-The part that you type,
-`ls -F /` in the second line of the example,
-typically has the following structure: a **command**,
-some **options** (also called **switches** or **flags**) and an **argument**.
-Flags start with a single dash (`-`) or two dashes (`--`), and change the behaviour of a command.
-Arguments tell the command what to operate on (e.g. files and directories).
-Sometimes options and arguments are referred to as **parameters**.
-A command can be called with more than one option and more than one argument: but a
-command doesn't always require an argument or an option.
-
-In the second line of the example above, our **command** is `ls`, with an **option** `-F` and an
-**argument** `/`. Each part is separated by spaces: if you omit the space
-between `ls` and `-F` the shell will look for a command called `ls-F`, which
-doesn't exist. Also, capitalization matters: `LS` is different from `ls`.
-
-Next we see the output that our command produced. In this case it is a listing
-of files and folders in a location called `/` - we'll cover what all these mean
-later today. Those using a macOS might recognize the output in this example.
-
-Finally, the shell again prints the prompt and waits for you to type the next
-command.
-
 In the examples for this lesson, we'll show the prompt as `$ `. You can make your
 prompt look the same by executing the command `PS1='$ '`. But you can also leave
 your prompt as it is - often the prompt includes useful information about who and where
 you are.
 
-Open a shell window and try executing `ls -F /` for yourself (don't forget that spaces
-and capitalization are important!). You can change the prompt too, if you like.
-
-### How does the shell know what `ls` and its options mean?
+So let's try our first command, which will list the contents of the current directory
+(assuming that our current directory contains some files or subdirectories):
```

Comment:
Nitpick: is the parenthetical necessary? I mean, if I run `ls` on an empty directory, does it not output the contents of that directory?

Reply:
Yep, fair point!